Advanced-Recruit-Training/src/LogFileReader.cpp

```diff
@@ -34,38 +32,43 @@ bool LogFileReader::readAll(const QString& fileName)
     {
         QString line = input.readLine();
         BatteryData batteryData;
-        if (!parseLine(line, batteryData))
+        if (!parseLine(line, batteryData))  //! taken before parseline out
```

Comment:
This function wasn't supposed to be edited at all. This goes for in the future too, try not to edit code that isn't related to the feature being currently worked on, doing so messes with diffs and stuff in version control.